mycroft/messagebus/send.py

```diff
@@ -38,17 +38,17 @@ def main():
         try:
             dataToSend = json.loads(sys.argv[2])
         except BaseException:
-            print "Second argument must be a JSON string"
-            print "Ex: python -m mycroft.messagebus.send speak " \
-                "'{\"utterance\" : \"hello\"}'"
+            print("Second argument must be a JSON string")
+            print("Ex: python -m mycroft.messagebus.send speak "
+                  "'{\"utterance\" : \"hello\"}'")
             exit()
     else:
-        print "Command line interface to the mycroft-core messagebus."
-        print "Usage: python -m mycroft.messagebus.send message"
-        print "       python -m mycroft.messagebus.send message JSON-string\n"
-        print "Examples: python -m mycroft.messagebus.send mycroft.wifi.start"
-        print "Ex: python -m mycroft.messagebus.send speak " \
-            "'{\"utterance\" : \"hello\"}'"
+        print("Command line interface to the mycroft-core messagebus.")
+        print("Usage: python -m mycroft.messagebus.send message")
+        print("       python -m mycroft.messagebus.send message JSON-string\n")
+        print("Examples: python -m mycroft.messagebus.send mycroft.wifi.start")
+        print("Ex: python -m mycroft.messagebus.send speak "
```

Comment:
For Python 2 compatibility here do we need to `from __future__ import print_function`?

Reply:
Not when making python2 legacy code compatible with python3.

If we were backporting from python3 where python3 features had been used we'd need to use `print_function`.

Reply:
But I'll add it to be on the safe side

Reply:
Oh yeah, only really a problem with something like `print('var', var)`.